set-dotnet-versions.bat

```diff
@@ -1,46 +1,58 @@
-
 REM Set these version variables and run the script to change all the files which need version updates
 
-REM The site extension version
+REM The site extension release version
 set major=0
-set minor=3
-set patch=15
+set minor=4
+set patch=0
 set version_postfix=-prerelease
 
-REM The agent version
+REM Specialized version for development package, increment as necessary for testing
+set development_major=100
+
+REM The agent version to deploy
 set agent_version=7.25.0
 
-REM The dotnet tracer version
-set tracer_version=1.21.1
+REM The dotnet tracer version to deploy
+set tracer_version=1.22.0
 
-REM *************************************************************
-REM All of the below code updates versions in files, do not touch
-REM *************************************************************
+REM **************************************************************************************************************************
+REM All of the below code updates versions in files, do not touch unless you wish to modify the structure of those files
+REM **************************************************************************************************************************
 
 set path_regex=[0-9]+_[0-9]+_[0-9]+
 set path_replacement=%major%_%minor%_%patch%
 
 set gitlab_yml=.gitlab-ci.yml
 powershell -Command "(gc .\%gitlab_yml%) -replace '%path_regex%', '%path_replacement%' | Out-File -encoding ASCII .\%gitlab_yml%"
+echo/
```

Comment:
Silly question, what are these `echo` commands for?

Reply:
It's a common way to write an empty line.
It doesn't really enhance the readability though so I'll just remove them.